core/src/main/java/org/codice/imaging/nitf/core/common/impl/AbstractSegmentWriter.java

```diff
@@ -70,9 +69,9 @@ private String padStringToLength(final String s, final int length) {
         return String.format("%1$-" + length + "s", s);
     }
 
-    private String hyphenPadStringToLength(final String s, final int length) {
+    private String hyphenPadStringToLength(final String s) {
         StringBuilder builder = new StringBuilder(s);
-        while (builder.length() < length) {
+        while (builder.length() < CommonConstants.STANDARD_DATE_TIME_LENGTH) {
```

Comment:
I'm guess SonarLint said that every time this method was invoked, we were passing in a constant as a parameter. SonarLint said to just inline the constatnt.

SonarLint might have made the wrong call. This method can pad any string to any length. It is a utility method. I'd be fine leaving it the way it was.

If we leave it as a single parameter method, I'd be in favor or renaming it to "hyphenPadDateTime". Or something like that.

Reply:
I definitely see the point. Sonar suggested this change for 3 of these helper methods, but this is the only one I applied it to if I recall correctly. I was on the fence before and can revert this change in case there is a need to pass in an alternate length.